services/backend/src/components/profiles/profiles.hooks.js

```diff
@@ -20,6 +23,26 @@ const includeSkills = () => (context) => {
 
 const liftProfilesSkills = () => (context) => {
     context.result = Profile.liftProfilesSkills(context.result);
+};
+
+const addSkills = () => async (context) => {
+    const {skills} = context.data;
+    const skillIds = skills.map(({id}) => id);
```

Comment:
This section should actually be wrapped in a conditional check on `skills`, otherwise it would break the normal creation of `profiles` because it would throw an `undefined` error for `map` when `skills` doesn't exist.. i.e.

```suggestion
    if (skills) {
        const skillIds = skills.map(({id}) => id);

        const sequelizeClient = context.app.get("sequelizeClient");
        const SkillsModel = sequelizeClient.models[tableNames.SKILLS];

        const hydratedSkills = await SkillsModel.findAll({
            where: {
                id: skillIds
            }
        });

        if (hydratedSkills) {
            const profile = context.result;
            await profile.addSkills(hydratedSkills);
        }
    }
```